### PR TITLE
feat: Add mlab database & Update readme

### PR DESCRIPTION
**Pull request by VernL**
- Add get and post routes
- Add .env file with database credentials
- Add .env to .gitignore
- Add instructions in README
- Update readme

### CODE REVIEW

File: backend/models/index.js [hunk 1]
```diff
@@ -1,72 +1,9 @@
-var mongoose = require('mongoose')
-var Bookstore = require('./bookstore')
+const mongoose = require('mongoose')
+require('dotenv').config()
```
Comment (RichardLitt) [inline backend/models/index.js]: Write in the README what should go in the .env file?

File: backend/models/index.js [hunk 2]
```diff
@@ -1,72 +1,9 @@
...
-  '\n' +
-  'In short, if you\'re in the area, pop in, peruse, and don\'t expect to leave without a book. Or three.',
-  thumbnail: 'img/book6.jpg',
-  websiteUrl: 'https://www.drawnandquarterly.com/',
-  locationUrl: 'https://goo.gl/maps/Sqpg8Tsa4Fp',
-  events: true,
+mongoose.set('debug', true)
+mongoose.connect(process.env.MONGODB_URI, {useMongoClient: true})
```
Comment (RichardLitt) [inline backend/models/index.js]: Sweet. Can you add instructions on setting this up in the README?

File: backend/routes/bookstores.js [hunk 3]
```diff
@@ -1,9 +1,71 @@
...
+    stores = await db.Bookstore.find()
+  } catch (err) {
+    throw new Error('Error finding entries in database')
+  }
+  return stores
+}
+
+async function addToDB (newStore) {
+  try {
+    await db.Bookstore.create(newStore)
```
Comment (RichardLitt) [inline backend/routes/bookstores.js]: Does Bookstore not have a .err method?

File: backend/routes/bookstores.js [hunk 4]
```diff
@@ -1,9 +1,71 @@
...
+                    newStore.coffee,
+                    newStore.wifi,
+                    newStore.plugs,
+                    newStore.events
+                  )
+  return store
+}
+
+class Bookstore {
+  constructor (name, summary, review, thumbnail, image, websiteUrl, locationUrl, coffee, wifi, plugs, events) {
```
Comment (RichardLitt) [inline backend/routes/bookstores.js]: This can't be the most efficient way? Is it?

File: README.md [hunk 5]
```diff
@@ -8,6 +8,10 @@ Starter pack for Bootstrap 4 Beta
 Includes complete Bootstrap 4.0.0-beta dev environment with gulp and sass
 Version: 1.0.0
 
+Persistent data stored in a MongoDB database
+To connect a database, create a .env file in the root directory and assign the database's URI connection scheme to a variable called MONGODB_URI 
+example: MONGODB_URI=mongodb://[username:password@]host1[:port1][,host2[:port2],...[,hostN[:portN]]][/[database][?options]]
```
Comment (RichardLitt) [inline README.md]: Don't I need to run mongod?

Comment (VernL) [reply README.md]: In our case, the database is hosted on mlab. However, we do need to have the backend running to connect. I'll add these two pieces of info.

**Pull request merged**